tq/transfer_queue.go

```diff
@@ -283,23 +283,44 @@ func (q *TransferQueue) enqueueAndCollectRetriesFor(batch batch) (batch, error)
 	tracerx.Printf("tq: sending batch of size %d", len(batch))
 
 	q.meter.Pause()
-	bRes, err := Batch(q.manifest, q.direction, q.remote, batch.ToTransfers())
-	if err != nil {
-		// If there was an error making the batch API call, mark all of
-		// the objects for retry, and return them along with the error
-		// that was encountered. If any of the objects couldn't be
-		// retried, they will be marked as failed.
-		for _, t := range batch {
-			if q.canRetryObject(t.Oid, err) {
-				q.rc.Increment(t.Oid)
-
-				next = append(next, t)
-			} else {
-				q.wait.Done()
-			}
+	var bRes *BatchResponse
+	if q.manifest.standaloneTransferAgent != "" {
```

Comment:
What do you think about moving this check into the `Batch()` function? That way `standaloneTransferAgent` just works for all `tq.Batch()` callers.